tutorials/getting_started/semantic_parsing.md

```diff
@@ -0,0 +1,315 @@
+# The AllenNLP Semantic Parsing Framework
+
+_Semantic parsing_ is the task of mapping language to some kind of formal meaning representation.
+This meaning representation could be a logical statement in lambda calculus or lambda-DCS, a set of
+instructions for a robot to follow, or even a python, java, or SQL program.  In some cases these
```

Comment:
I agree with Kevin that having running examples would be very useful. You can start here, may be with two examples, one from SQL, and another from a domain specific language like the one for Wikitables.

Reply:
Would also help to (semi-)formally define the inputs and outputs of a semantic parser, while training and testing. That would define frequently used terms like "utterance", "logical form", and "actions".